### DISCUSSION TITLE
EasyNLP updates 220512

### DISCUSSION THREAD

**Discussion by minghui** [alibaba/EasyNLP#84] [Announcements]
EasyNLP updates 220512

New features:
- Add CLUE benchmark example: https://github.com/alibaba/EasyNLP/tree/master/benchmarks/clue
- CPU training of sequence classification: https://github.com/alibaba/EasyNLP/issues/59
- BLIP text-image retrieval support: https://github.com/alibaba/EasyNLP/issues/40
- CLIP text-image retrieval support: https://github.com/alibaba/EasyNLP/issues/39

Updates
- fix MetaKD issue: https://github.com/alibaba/EasyNLP/issues/80
- fix optimizer amp issue: https://github.com/alibaba/EasyNLP/issues/64

**Comment by wenmengzhou** [comment]
great work
